<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationForm</class>
 <widget class="QDialog" name="RegistrationForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegistrationForm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>180</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>381</width>
        <height>154</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QLineEdit" name="loginLineEdit"/>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="positionCBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="surnameLineEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Имя:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="passwordLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="loginLabel">
         <property name="text">
          <string>Логин:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string>Пароль:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="positionLabel">
         <property name="text">
          <string>Должность:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="surnameLabel">
         <property name="text">
          <string>Фамилия:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="passRepeatLabel">
         <property name="text">
          <string>Повторите пароль:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="passRepeatLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="2" rowspan="2">
        <widget class="QPushButton" name="showHideBtn">
         <property name="text">
          <string>Показать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="registrationBtn">
       <property name="text">
        <string>Зарегистрироваться</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>surnameLineEdit</tabstop>
  <tabstop>positionCBox</tabstop>
  <tabstop>loginLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>passRepeatLineEdit</tabstop>
  <tabstop>registrationBtn</tabstop>
  <tabstop>showHideBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>registrationBtn</sender>
   <signal>clicked()</signal>
   <receiver>RegistrationForm</receiver>
   <slot>registrationSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>RegistrationForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showHideBtn</sender>
   <signal>clicked()</signal>
   <receiver>RegistrationForm</receiver>
   <slot>showHideSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>registrationSlot()</slot>
  <slot>showHideSlot()</slot>
 </slots>
</ui>
